scrapers/scrape_ag.sh

```diff
@@ -23,3 +23,5 @@ print('ICU:', sc.find(r'([0-9]+) Person(en)? werden auf Intensivstationen behand
 print('Vent:', sc.find(r'([0-9]+) Person(en)? k(ü|&uuml;)nstlich beatmet werden', d))
 
 print('Deaths:', sc.find(r'([0-9]+) Person(en)? an den Folgen des Coronavirus ?verstorben', d))
+
+print('Source:', 'https://www.ag.ch' + sc.find(r'<a .*?href="(.+?)".*?>.*?Lagebulletin.*?</a>', d))
```

Comment:
We already have everything in place to gather URLs that are being used in the scraper, instead of introducing a new key `Source:` that needs extra parsing, I'd suggest to change this line to

```
print('Downloading:', 'https://www.ag.ch' + sc.find(r'<a .*?href="(.+?)".*?>.*?Lagebulletin.*?</a>', d))
```

And everything else is already taken care of in the parser.

Reply:
Okay, makes sense - updated!